.travis.yml

```diff
@@ -1,6 +1,9 @@
 sudo: false
 
 rvm:
-  - "2.0"
-  - "2.1"
-  - "2.2"
+  - 2.0
```

Comment:
It is preferable to keep those as strings so that they're not parsed as floats in the YAML file.

Reply:
Updated